src/lib/components/button.svelte

```diff
@@ -66,8 +67,9 @@
   {id}
   {type}
   {name}
-  class="{px} {py} {ts} {border} {text} {extraClass} {background} whitespace-nowrap rounded leading-normal outline-none hover:underline focus:shadow-focus"
+  class="{px} {py} {ts} {border} {text} {extraClass} {background} whitespace-nowrap rounded leading-normal outline-none focus:shadow-focus"
   class:w-full={wFull}
+  class:hover-underline={hoverUnderline}
```

Comment:
```suggestion
  class:hover:underline={hoverUnderline}
```
et tu n'as plus besoin de definir la class hover-underline.
Mais est-ce qu'on ne voudrait pas activer ça pour tous les linkButtons en fait? A voir avec Sylvain…

Reply:
Dans certains, le comportement au survol est un changement de _background_ donc je peux déjà te dire que non ^^
D'ailleurs, j'ai ajouté l'option après que Sylvain ait rejeté le soulignement partout :)

Reply:
Et sinon je ne pensais pas que le `class:hover:underline` fonctionnait 👍